.github/workflows/ci.yml

```diff
@@ -80,7 +80,7 @@ jobs:
           components: 'native-image'
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Test
-        run: mvn --batch-mode --no-transfer-progress -P native test
+        run: mvn --batch-mode --no-transfer-progress -P native verify
```

Comment:
why the change ?

Reply:
to include the `integration-test` maven lifecycle, which runs later in the process than `test`.

Reply:
the issue i see is that `verify` will also run spotless, which is already ran in a separate CI job. Can we run `test integration-test` ? Do we even need to run `test` for the native image ? Can't we just run `integration-test` only ?

Reply:
spotless doesn't seem to be registered to run in any default maven phase, so unless you call it directly (such as in the CI) it doesn't run. So `verify` should be fine, as it should be functionally the same as targetting `integration-test` only.

`test` will need to be executed, adding `-DskipTests` will also skip the native tests.